support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.jsx

```diff
@@ -292,7 +294,13 @@ function WeeklyCheckoutForm(props: PropTypes) {
             submissionError={props.submissionError}
           />
           <FormSection noBorder>
-            <Button aria-label={null} type="submit">Continue to payment</Button>
+            <Button
+              id="qa-submit-button"
+              aria-label={null}
```

Comment:
I thought this was going to become a default value in Button?

Reply:
I've just left it for now because once we have PayPal on the other checkouts we can get rid of it entirely

Reply:
Oh yes, forgot about that! I'll do that now